examples/wealth_distribution.jl

```diff
@@ -117,7 +117,7 @@ Random.seed!(5) # hide
 init_wealth = 4
 model = wealth_model_2D(; wealth = init_wealth)
 adata = [:wealth, :pos]
-data, _ = run!(model, agent_step!, 10; adata = adata, when = [1, 5, 9])
+data, _ = run!(model, agent_step_2d!, dummystep, 10; adata = adata, when = [1, 5, 9])
```

Comment:
Oh good catch

Reply:
Yeah! I'm sure I've looked at this example 1000 times and didn't notice it.